frontend/src/views/housing/project/ProjectView.vue

```diff
@@ -346,6 +385,11 @@ a {
   text-decoration: underline;
 }
 
+.disclaimer-block {
+  outline: solid 1px $app-grey;
+  border-radius: 8px;
```

Comment:
Style: Some of these can be re-defined in rem units
```css
  outline: solid 0.063rem $app-grey;
  border-radius: 0.5rem;
```
Unless otherwise specified, relative calculated units should be preferred so that high resolution displays can appropriately scale as needed.

Reply:
swapped all `px` values to `rem`